cepimose/parser.py

```diff
@@ -167,6 +167,9 @@ def get_manufacturer(num):
             value = C[1]
             obj = create_obj(date)
             obj[manufacturer] = value
+            print(parsed_data[-1], manufacturer, value, sep="\t")
+            setattr(parsed_data[-1], manufacturer, value)
+            print(parsed_data[-1], manufacturer, value, sep="\t")
```

Comment:
Are these prints intended to stay?

```suggestion
            setattr(parsed_data[-1], manufacturer, value)
```

Reply:
Definitely not.